setup.py

```diff
@@ -495,6 +495,7 @@ def package_tree(pkgroot):
 
 package_path(join(SERVER, 'static'))
 package_path(join(ROOT, 'bokeh', 'core', '_templates'))
+package_path(join(ROOT, 'bokeh', 'server', 'views'), ('.html'))
```

Comment:
@bryevdv looking at this patch again after some sleep, is it a better fix to move bokeh/server/views/app_index.html into bokeh/server/static/html?  It would also require a oneline change to the server code.  I wouldn't mind redoing this if you think that's better.